runners/spark/src/main/java/org/apache/beam/runners/spark/translation/EvaluationContext.java

```diff
@@ -129,10 +128,6 @@ public void putDataset(PValue pvalue, Dataset dataset) {
     datasets.put((PValue) getOutput(transform), new UnboundedDataset<>(values, jssc, coder));
   }
 
-  void putPView(PValue view, Iterable<? extends WindowedValue<?>> value) {
```

Comment:
Why not keep `put/get` encapsulated here ? saves the ugly `context.getPviews().putPViews()`